### ISSUE TITLE
Post-match follow up actions

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#18]
As a primary user, I can click on a recommendation in order to either view the relevant profile, or immediately send a “notification” to the relevant primary user, so that I can either evaluate the recommendation or immediately contact the appropriate person.

TBD what "notification" means.

**Comment by LucyJW** [comment]
For the MVP, we could simply display a call to action to email/call the member with a quick link to the member's contact details (rather than a notification being sent through the platform).

**Comment by asimonw** [comment]
![fullsizerender_4](https://cloud.githubusercontent.com/assets/15235804/17745470/c1e29f3a-64a3-11e6-923b-cb4de4359781.jpg)

Continued from #14. When clicking on one of the matches, the user is presented with two options:
* a simple link to the respective org profile
* send a notification to the primary contact of that org (initially, the notification could be fully automated, standard text, with title indicating the relevant challenge)

**Comment by JMurphyWeb** [comment]
@asimonw we do not currently have notifications in priority 2 or 3. With that in mind, I don't know if you want us to do anything on this issue still?

**Comment by asimonw** [comment]
@JMurphyWeb No need to do anything on this before finishing all priority 2 issues. When a user clicks on a suggested match they should be taken to the corresponding org profile screen. I'm assuming that this is still the current behaviour?

**Comment by JMurphyWeb** [comment]
yeah this is still the same. I just wanted to see if there was any work to be done on it.
